packages/material-ui-lab/src/CalendarPicker/PickersCalendar.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
```

Comment:
Include `PickersCalendar` here because of style override (`min-height`) in `DateRangePickerViewDesktop`